root/etc/e-smith/templates/etc/ssh/sshd_config/70Restricted2Sftp

```diff
@@ -0,0 +1,23 @@
+{
+    # restrict groups to sftp, ssh access is refused
+    my $domain = $DomainName || die ('Cannot retrieve DomainName');
+    my @SftpRestricted = ();
+
+    foreach ( split(',',$sshd{'AllowGroups'} || '')) {
+        my ($group, $sftp) = split(':', $_);
+        if(!$group || !$sftp) {
+            next;
+        }
+        $group .= "\@$domain" if ($group !~ '@');
+        push @SftpRestricted, $group if ($sftp eq 'sftp');
+    }
+
+    if (@SftpRestricted) {
+        my $matchGroup = join (",", map { $_ } @SftpRestricted);
+        $OUT .= "Match Group \"$matchGroup\"\n";
```

Comment:
The double quotes around `$matchGroup` have a different implementation, compared to what happens in [line 31](https://github.com/NethServer/nethserver-openssh/pull/8/files#diff-5f032d38fa07cb287a0402ad33a4730dR31).

Do we need to protect whitespace in the group name in the same way as above?

Reply:
no the way to protect group name against space for sftp is completely different from AllowGroups, for example it is a comma separated list, and not a space separated list like you see above. If we do not use comma and protect against spaces, the service crash